window_size: 1024
noise_floor_buffer_length: 32
source_loc_buffer_length: 8
similarity_distance: 30
noise_floor_sensitivity: 0.8
noise_threshold_db: 65